# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/ContentParent.h
# Commit: 4b2f994f3fe2
# Full Hash: 4b2f994f3fe2cad3895cc4d5044a3466a2d5eda4
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2020-10-12 15:27:19
# Regressor Bug: 1668253
# File Overlap Count: 1
# Description:
#   Bug 1668253 - Convert hyphenation from ipc::SharedMemoryBasic to base::SharedMemory, and freeze blocks before sharing to content processes. r=heycam
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D91945
# ==============================================================================

diff -r bc5b6dcb7057 -r 4b2f994f3fe2 dom/ipc/ContentParent.h
--- a/dom/ipc/ContentParent.h	Mon Oct 12 10:23:27 2020 +0000
+++ b/dom/ipc/ContentParent.h	Mon Oct 12 10:24:06 2020 +0000
@@ -1204,7 +1204,7 @@
       const mozilla::fontlist::Pointer& aFamilyPtr);
 
   mozilla::ipc::IPCResult RecvGetHyphDict(
-      nsIURI* aURIParams, mozilla::ipc::SharedMemoryBasic::Handle* aOutHandle,
+      nsIURI* aURIParams, base::SharedMemoryHandle* aOutHandle,
       uint32_t* aOutSize);
 
   mozilla::ipc::IPCResult RecvNotifyBenchmarkResult(const nsString& aCodecName,